components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-list-load.jsp

```diff
@@ -38,9 +37,12 @@
     String BUNDLE = "org.wso2.carbon.idp.mgt.ui.i18n.Resources";
     ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE, request.getLocale());
     String callback = request.getParameter("callback");
-    if (StringUtils.isBlank(callback) || !IdentityValidationUtil
-            .isValidOverBlackListPatterns(callback, IdentityValidationUtil.ValidatorPattern.URI_RESERVED_EXISTS
-                    .name())) {
+    
+    final String trustedCallbackEdit = "idp-mgt-edit.jsp";
```

Comment:
are we following the coding conventions here to define a constant?

Reply:
will be fixed with https://github.com/wso2/carbon-identity-framework/pull/2125/commits/ba290f575bee595f8af3594f93b5a4c0e7fd2f4c